### ISSUE TITLE
Package Deprecated in error: Microsoft.AspNetCore.DataProtection.SystemWeb

### ISSUE THREAD

**Issue by RWebster-Noble** [dotnet/core#8900]
### Package Id

Microsoft.AspNetCore.DataProtection.SystemWeb

### Package Version(s)

2.2.0

### Additional details

We are using the Microsoft.AspNetCore.DataProtection.SystemWeb package in our .NET Framework projects.

In reviewing package updates we can see that the latest version of this package is deprecated but there appears to be no later version or any clear alternatives or upgrade path available. Therefore I believe that package is deprecated is in error and should receive future updates.

The [package deprecation announcement](https://github.com/dotnet/announcements/issues/217) states "AspNetCore 2.2 is a unique category, particularly around .NET Framework usage. We will work out how best to address these packages getting caught up and staying current at each release." but this package appears to have fallen through the cracks as the DataProtection Api is definitely not a deprecated product. In-fact there is current ASP.NET Core 7.0 documentation instructing to use this package here: [Replace the ASP.NET machineKey in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/security/data-protection/compatibility/replacing-machinekey?view=aspnetcore-7.0).

**Comment by leecow** [comment]
@mkArtakMSFT @wtgodbe

**Comment by wtgodbe** [comment]
The latest in-support version of that package is 2.1.1: https://www.nuget.org/packages/Microsoft.AspNetCore.DataProtection.SystemWeb/2.1.1. We recommend using 2.1 rather than 2.2 as it is still in support. As for the SystemWeb library, @adityamandaleeka is that still around in .NET 8.0?

**Comment by RWebster-Noble** [comment]
@adityamandaleeka @mkArtakMSFT @wtgodbe

Any updates on this?
Is it still recommended to use 2.1?

**Comment by wtgodbe** [comment]
Yes, we recommend using 2.1. We're working on improving the dev experience in VS so that 2.2 packages don't get recommended as updates anymore.
